plugin/lighthouse/handler_test.go

```diff
@@ -290,11 +308,12 @@ func testClusterStatus() {
 			mockCs.clusterStatusMap[clusterID] = false
 			mockCs.clusterStatusMap[clusterID2] = false
 		})
-		It("should return RcodeServerFailure", func() {
+		It("should return empty response (NODATA)", func() {
 			executeTestCase(lh, rec, test.Case{
-				Qname: service1 + "." + namespace1 + ".svc.cluster.local.",
-				Qtype: dns.TypeA,
-				Rcode: dns.RcodeServerFailure,
+				Qname:  service1 + "." + namespace1 + ".svc.cluster.local.",
```

Comment:
Why are you testing cluster.local instead of multicluster/supercluster/etc?

Reply:
Same question for others

Reply:
plugin code doesn't really care as `supercluster.local` is more of a configuration parameter. Since this is handler UT, configuration doesn't apply to it. Agree that should change this, was waiting for `clusterset` to be formalized before making this change.